eldritch/test_combat.py

```diff
@@ -1978,5 +1978,108 @@ def testDeactivatesOnCombatLoss(self):
     self.assertTrue(witch.has_attribute("magical resistance", self.state, self.char))
 
 
+class BindMonsterTest(EventTest):
+  def setUp(self):
+    super().setUp()
+    self.bind_monster = items.BindMonster(0)
+    self.char.possessions = [self.bind_monster]
+
+  def start(self, monster):
+    # pylint: disable=attribute-defined-outside-init
+    self.combat = Combat(self.char, monster)
+    self.state.event_stack.append(self.combat)
+
+    with mock.patch.object(
+        events.random, "randint", new=mock.MagicMock(return_value=5)
+    ):
+      fight_or_flee = self.resolve_to_choice(MultipleChoice)
+      fight_or_flee.resolve(self.state, "Fight")
+
+      return self.resolve_to_choice(CombatChoice)
+
+  def testSanePass(self):
+    self.char.sanity = 4
+    self.char.fight_will_slider = 0
+    worm = monsters.GiantWorm()
+    self.start(worm)
+    self.assertEqual(self.char.sanity, 3)
+    self.assertEqual(self.char.stamina, 5)
+    self.state.event_stack.append(self.state.usables[0]["Bind Monster0"])
+    choose_ignore = self.resolve_to_choice(SpendChoice)
+    self.spend("sanity", 2, choose_ignore)
+    choice = self.resolve_to_choice(SpendChoice)
+    choice.resolve(self.state, "Cast")
+    with mock.patch.object(
+        events.random, "randint", new=mock.MagicMock(return_value=5)
+    ):
+      choice = self.resolve_to_choice(CombatChoice)
```

Comment:
Oh, this is kind of weird. You successfully cast the spell, but then you have to resolve the combat choice? I think you want the spell to cancel both the CombatChoice and the CombatRound, (otherwise, you'll end up making a combat check again, and be asked if you want to spend clue tokens, etc.).